cmake/Modules/FindZstd.cmake

```diff
@@ -0,0 +1,28 @@
+# Find Zstd library
+#
+#  Zstd_INCLUDE_DIRS - where to find zstd.h, etc.
+#  Zstd_LIBRARIES    - List of libraries when using libzstd
+#  Zstd_FOUND        - True if libzstd is found.
+
```

Comment:
zstd has a pkg-config file, so you use that as a hint.

```
find_package(PkgConfig)

pkg_check_modules(PC_ZSTD libzstd)
find_path(Zstd_INCLUDE_DIR NAMES zstd.h HINTS ${PC_ZSTD_INCLUDE_DIRS})
find_library(Zstd_LIBRARY NAMES zstd HINTS ${PC_ZSTD_LIBRARY_DIRS})
```

Or just use pkg-config without a `FindZstd.cmake` at all:
```
find_package(PkgConfig REQUIRED)
pkg_check_modules(Zstd REQUIRED IMPORTED_TARGET libzstd)
```
That will create a `PkgConfig::Zstd` target.

Reply:
@junghans I adapted the last variant. Note, that now the CMake report only lists PkgConfig as added, not Zstd. This probably is also true with other packages that use this mechanism.